pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -284,12 +285,22 @@ func UpdateKubernetesObjects(name string, service kobject.ServiceConfig, objects
 		template.Spec.Containers[0].WorkingDir = service.WorkingDir
 		template.Spec.Containers[0].VolumeMounts = volumesMount
 		template.Spec.Volumes = volumes
-		// Configure the container privileged mode
+
+		securityContext := &api.SecurityContext{}
 		if service.Privileged == true {
-			template.Spec.Containers[0].SecurityContext = &api.SecurityContext{
-				Privileged: &service.Privileged,
+			securityContext.Privileged = &service.Privileged
+		}
+		if service.User != "" {
+			uid, err := strconv.ParseInt(service.User, 10, 64)
+			if err != nil {
+				logrus.Warn("Ignoring User directive. User has to specfied as uid (numeric).")
```

Comment:
Just a little spelling error :)

`Ignoring user directive. User to be specified as a UID (numeric).`